starport/internal/version/version.go

```diff
@@ -3,14 +3,15 @@ package version
 import (
 	"fmt"
 	"runtime"
+	"time"
 )
 
 var (
 	// Version is the semantic version of Starport.
-	Version = "dev"
+	Version = "0.12.0-develop"
```

Comment:
Thank you @lukitsbrian, I was looking to have this feature for a while as well.

When official [releases](https://github.com/tendermint/starport/releases) are used, `version` command should show the current release and date but we didn't have this for development binaries.

Would you like to pass version info and date through our Makefile's [build](https://github.com/tendermint/starport/blob/0da9910/Makefile#L6) section via _ldflags_ so that way we avoid the need of maintaining the version info here by hand?

Btw, _Date_ you've provided seems like a runtime value which will be different every time we run the binary. Passing the date through _ldflags_ should also fix it.

Reply:
Can we pass git object hash in development, so it's easier to know which development version you are currently using?

Reply:
Sounds good! Moved this to `Makefile`.

Current output looks like this:
```
$ starport version
starport version v0.12.0-71-g35e37fe darwin/amd64 -build date: 2020-12-07T00:54:33
git object hash: 35e37fe5f767cc9ec60ab126837512b6188aa353
```

When `Head` is omitted (eg. [here](https://github.com/tendermint/starport/blob/0da99107e1403762768f40d020cd73a2e0dc50f7/.goreleaser.yml#L4)) it shouldn't output the `git object hash`.